### ISSUE TITLE
Adapt docstring generation regarding inheritance

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/autowrap#180]
E.g. instead of "Inherits from `ClassXYZ`" one could use sphinx :py:class:`ClassXYZ` rst syntax.

Also, the inherited methods should be grouped at the beginning or end and maybe noted with "inherited from"?

Currently they are listed in alphabetical order.

**Comment by jpfeuffer** [comment]
Yes thanks. You need to setup both autowrap and some sphinx autodoc documentation for the module that you are testing to see how your autowrapped module gets documented.

**Comment by khushiiagrawal** [comment]
> Yes thanks. You need to setup both autowrap and some sphinx autodoc documentation for the module that you are testing to see how your autowrapped module gets documented.

Okay. Will do it. Thanks!
